### ISSUE TITLE
Add custom repositories.yaml support for Helm repositories

### ISSUE THREAD

**Issue by kangshung** [lensapp/lens#2875]
**What would you like to be added**:
I would like to have an option to either pick my own repositories.yaml with Helm repositories or paste my own config inside the app.

**Why is this needed**:
When I am on WSL, I use only its terminal and keep all the synced Helm repos within this filesystem rather than the Windows one.

Lens Apps section doesn't allow me to use my WSL Helm configuration
--> I cannot install apps with Lens from my always current repos (because they are WSL only and Lens cannot detect them)
--> every time I add/update a repo I need to synchronize the Windows version with it just for Lens
--> it makes it unusable for me

Switching to Windows makes no sense to me.

Would be awesome if I had an option to choose a path for this file the same way I can choose path to kubectl binary etc. 

**Environment you are Lens application on:**
- Kubernetes distribution: doesn't matter
- Desktop OS: Windows

**Comment by Nokel81** [comment]
You are already able to add your own custom repos. I guess you are asking to be able to add them in bulk, right?

**Comment by kangshung** [comment]
@Nokel81 Yes, sync with the file. Helm binary uses a file that keeps the state of current repositories.

https://github.com/helm/helm/blob/main/cmd/helm/testdata/repositories.yaml

      --repository-config string    path to the file containing repository names and URLs (default "~/.config/helm/repositories.yaml")
